src/workloads/c2c/MongosyncScripts.yml

```diff
@@ -16,31 +16,42 @@ StartMongosync:
   Repeat: 1
   Command: python3
   MetricsName: StartMongosync
-  ScriptPath: ./src/genny/src/workloads/c2c/mongosync_helpers.py -m start
+  Script: |
+    import mongosync_actor as actor
```

Comment:
I like this better. 

It's probably diminishing returns, but we could likely also just make this script an endpoint of the module.